arb_os/errorHandler.mini

```diff
@@ -66,6 +70,14 @@ public view write func errorHandler() {
     } else {
         // Application code must have hit an error.
         inErrorHandler = false;
+        let _ = evmCallStack_returnFromCall(
```

Comment:
Now that `evmCallStack_returnFromCall` may resolve the error handler, the `debug((664, errInfo = 0));` from `evmCallStack_callHitError` may not occur. This shouldn't matter since we aren't consuming this on the node's side